tests/simulator/refinement/harris.py

```
@@ -0,0 +1,225 @@
+
+    def density(x, y):
+        from pyphare.pharein.global_vars import sim
+
+        Ly = sim.simulation_domain()[1]
+        return (
+            0.4
+            + 1.0 / np.cosh((y - Ly * 0.3) / L) ** 2
+            + 1.0 / np.cosh((y - Ly * 0.7) / L) ** 2
+        )
+
+    def S(y, y0, l):
```

Ruff

Ambiguous variable name: `l`

(E741)